GHermeneus/include/GHermeneus/Instruction.h

```diff
@@ -0,0 +1,95 @@
+//
+// Created by Jelle Spijker on 7/31/20.
+//
+
+#ifndef GCODEHERMENEUS_INSTRUCTION_H
+#define GCODEHERMENEUS_INSTRUCTION_H
+
+#include <string_view>
+#include <utility>
+#include <vector>
+
+#include "GCode.h"
+
+namespace GHermeneus
+{
+    /*!
+     * An dialect interpreted GCode line which consist of a corresponding line number, command key (e.q. G0, M104) and
+     * a vector of Parameters of type double.
```

Comment:
Comments mismatch: I see you parameterized the (type of the) ... eh ...  parameters, so it no longer _has_ to be double.